Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM alpine:3.3
 MAINTAINER Nicolas Degory <ndegory@axway.com>
 
+#Need to keep curl and bash after installation for containerPilot
 RUN apk update && \
-    apk --no-cache add python ca-certificates && \
-    apk --virtual envtpl-deps add --update py-pip python-dev curl && \
+    apk --no-cache add python ca-certificates curl bash && \
```

Comment:
@freignat91 let's keep curl by default, we need it also for customization of most of the images in the init script